x-pack/legacy/plugins/siem/public/containers/detection_engine/signals/api.ts

```diff
@@ -62,89 +58,65 @@ export const updateSignalStatus = async ({
   status,
   signal,
 }: UpdateSignalStatusProps): Promise<unknown> => {
```

Comment:
@spong I noticed this `unknown` here, and the fact that w're not using the return value. Should we just type this as `void` and not return anything, or should we update the return type here?

Reply:
IIRC I typed it as `unknown` as the API was still in flux while this was being developed. I'll be looping back over the Detection Engine API to rework bulk actions + import/export and can add a proper type then if that works for you.

Reply:
Sounds good!